src/core/Bot.py

```diff
@@ -343,9 +341,8 @@ async def get_or_fetch_message(
         after = discord.Object(message_id - 1)
         if fetch:
             async for msg in channel.history(limit=1, before=before, after=after):
-                if msg:
-                    self.message_cache[msg.id] = msg
-                    return msg
+                self.message_cache[msg.id] = msg
+                return msg
```

Comment:
It will be useless if checking the availability of `msg`. As it won't `__aiter__` if no message found